ec2utils/ec2deprecateimg/lib/ec2utils/ec2deprecateimg.py

```diff
@@ -0,0 +1,401 @@
+# Copyright 2015 SUSE LLC, Robert Schweikert
+#
+# This file is part of ec2deprecateimg.
+# ec2deprecateimg is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# ec2deprecateimg is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with ec2deprecateimg. If not, see <http://www.gnu.org/licenses/>.
+
+import boto
+import boto.ec2
+import ConfigParser
+import datetime
+import dateutil.relativedelta
+import re
+
+from ec2utils.ec2DepImgExceptions import *
+
+class EC2DeprecateImg:
+    """Deprecate EC2 image(s) by tagging the image with 3 tags, Deprecated on,
+       Removal date, and Replacement image."""
+
+    def __init__(self,account,
+                 depPeriod=6,
+                 config=None,
+                 depImgID=None,
+                 depImgName=None,
+                 depImgNameFrag=None,
+                 depImgNameMatch=None,
+                 publicOnly=None,
+                 region=None,
+                 replImgID=None,
+                 replImgName=None,
+                 replImgNameFrag=None,
+                 replImgNameMatch=None,
+                 verbose=None,
+                 virtType=None):
+
+        self.account    = 'account-' + account
+        self.depPeriod  = depPeriod
+        self.configFile = config
```

Comment:
Again, a bit confusing.  Neither `self.configFile` and `config` refers to the file handle or I/O stream; both refer to the file path.